parser_methods.py

```diff
@@ -404,7 +405,7 @@ def pdf_xref_objects_to_xml(cls, pdf_file):
         text = ''
         with open(pdf_file, 'rb') as fp:
             parser = PDFParser(fp)
-            doc = PDFDocument(parser, None)
+            doc = PDFDocument(parser)
```

Comment:
Is this b/c None is passed as the 2nd param by default?

Reply:
From [their docs](https://pdfminersix.readthedocs.io/en/latest/tutorial/composable.html), this looks correct. I'm not sure why `None` was needed before. 🤔